### ISSUE TITLE
Fano web application used Free Pascal 3.0.4 TFpHttpServer returns empty response with incorrect Content-Type

### ISSUE THREAD

**Issue by zamronypj** [fanoframework/fano#28]
Fano web application that use Free Pascal 3.0.4 TFpHttpServer returns empty response with incorrect Content-Type when compiled with  FPC 3.0.4

## How to reproduce

- Install FPC 3.0.4
- Create http project with latest Fano CLI 
```
$ fanocli --project-http=myproject
$ cd myproject && fanocli --controller=Home --route=/ 
$ ./build.sh
$ ./bin/app.cgi
```
- It will run http application in port 20477 
- Open `http://127.0.0.1:20477` in browser. Browser downloads response.

**Comment by zamronypj** [comment]
Fixed in 076846e5b96638f1

**Comment by zamronypj** [comment]
This issue is caused by different behavior of FileExists() in FPC 3.0.4 and FPC 3.2.0. Prior to FPC 3.2.0, In Unix/Linux, FileExists() returns true for both for existing file and directory. In FPC 3.2.0, FileExists() returns false if input name is existing directory.
